Client/Frontend/Browser/BrowserViewController/BrowserViewController+ZoomPage.swift

```diff
@@ -0,0 +1,57 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Shared
+
+extension BrowserViewController {
+    func updateZoomPageBarVisibility(visible: Bool) {
+        if visible {
+            if zoomPageBar == nil {
```

Comment:
nit, can be one `if`
```suggestion
        if visible, zoomPageBar == nil  {
```